README.md

```diff
@@ -73,6 +73,53 @@ The AET dictionary JSON can also be specified directly via the "--aet_dictionary
 
 For the list of command line flags, see [here](import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java)
 
+## Routing to multiple DICOM stores
+
+Import adapter C-STORE service supports routing to multiple DICOM stores based on tags in incoming instance and client AET.
```

Comment:
wording nit pick: "tags in the incoming instance and the client AE title."

Reply:
Ok